test/fft_tests/src/fft.c

```diff
@@ -28,6 +28,9 @@
 #include <string.h>
 #include "fft.h"
 
+#ifndef M_PI
+#define M_PI ( 3.14159265358979323846 )
+#endif
```

Comment:
I thought you've already added this defenition

Reply:
yes, but it was in `xmath.h`. I will remove this define and include the header file.